google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java

```diff
@@ -92,6 +94,7 @@ public class Publisher implements PublisherInterface {
   private static final Logger logger = Logger.getLogger(Publisher.class.getName());
 
   private static final String GZIP_COMPRESSION = "gzip";
+  private static final String OPEN_TELEMETRY_TRACER_NAME = "cloud.google.com/java/pubsub";
```

Comment:
I feel like this should be `com.google.pubsub.v1` since that seems to be the canonical package name?

Reply:
Done.